app/controllers/course/discussion/posts_controller.rb

```diff
@@ -0,0 +1,106 @@
+class Course::Discussion::PostsController < Course::ComponentController
+  before_action :set_topic
```

Comment:
So you mean I can change `set_conditional` to `conditional` and just return the object instead of setting `@conditional`?